posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,253 @@
+
+[#schemaproperty]
+==== @SchemaProperty
+
+The @SchemaProperty annotation has been added to allow you to include properties in the definition of a schema when using the @Schema annotation. Previously, the @Schema annotation did not allow you to define any properties. Properties were introspected from the implementation class.
+With the introduction of the @SchemaProperty annotation, a list of properties can now defined using the properties attribute on the @Schema annotation, as follows:
+
+[source, java]
+----
+@Schema(
+    name = "User",
+        description = "Telephone number to contact the user”,
+        implementation = User.class,
+        properties = {
+            @SchemaProperty(
+                name = "phone",
+                description = "Telephone number to contact the user”
+            )
+        }
+    )} 
+----
+[#requestbodyschema]
+==== @RequestBodySchema
+
+The @RequestBodySchema annotation has been added to provide a shorthand mechanism to specify the schema for a request body. Previously, defining the schema for a request body required the use of three different annotations, as follows:
+[source, java]
+----
+@RequestBody(
+    content = {
+        @Content(
+            schema = @Schema(
+                implementation = MyRequestObject.class
+            )
+        )
+    }
+)
+----
+
+With the introduction of the @RequestBodySchema annotation, this can be simplified to the following:
+
+[source, java]
+----
+@RequestBodySchema(MyRequestObject.class) 
+----
+
+[#apiresponseschema]
+==== @APIResponseSchema
+
+The @APIResponseSchema annotation has been added to provide a shorthand mechanism to specify the schema for a response body. Previously, defining the schema for a response body required the use of three different annotations, as follows:
+
+[source, java]
+----
+@APIResponse(
+    content = {
+            @Content(
+                schema = @Schema(
+                    implementation = MyResponseObject.class
+                )
+            )
+        }
+    )
+----
+
+With the introduction of the @RequestBodySchema annotation, this can be simplified to the following:
+
+[source, java]
+----
+@APIResponseSchema(MyResponseObject.class)
+----
+
+[#mpopenapischeme]
+==== mp.openapi.schema.*
+The mp.openapi.schema.* MicroProfile Config property prefix has been added to allow you to define the schema for Java classes using configuration rather than code. The use of this property is functionally equivalent to the use of the @Schema annotation on a Java class, but may be used in cases where you do not have access to the source code of a class. The remainder of the property key must be the fully-qualified class name and the value must be a valid OpenAPI schema object, specified in the JSON format.
```

Comment:
Put `mp.openapi.schema.*` in backticks (monospace)

Reply:
And for the annotation names throughout.

Reply:
Done